src/main/scala/org/apache/spark/sql/pulsar/PulsarMetadataReader.scala

```diff
@@ -248,6 +249,7 @@ private[pulsar] case class PulsarMetadataReader(
       case None =>
         throw new RuntimeException("Failed to get topics from configurations")
     }
+    topics
```

Comment:
Since the return value is not used anywhere. We can just use Unit as the return value instead.

```scala
  private def getTopics(): Unit = {
    val optionalTopics = caseInsensitiveParameters.find({case (key, _) => TopicOptionKeys.contains(key)})
    topics = optionalTopics match {
      case Some((TopicSingle, value)) =>
        TopicName.get(value).toString :: Nil
      case Some((TopicMulti, value)) =>
        value.split(",").map(_.trim).filter(_.nonEmpty).map(TopicName.get(_).toString)
      case Some((TopicPattern, value)) =>
        getTopics(value)
      case None =>
        throw new RuntimeException("Failed to get topics from configurations")
    }
  }
```